### PR TITLE
fix: apply reference value with usage unit in 3 widgets

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### CODE REVIEW

File: apps/web/src/services/dashboards/widgets/_helpers/__tests__/widget-value-label-helper.test.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        it('should return value if referenceIdKey is included in allReferenceTypeInfo but referenceMap is not provided', () => {
+            expect(getWidgetValueLabel('aws', { allReferenceTypeInfo: mockAllReferenceTypeInfo, referenceIdKey: 'project_id' })).toBe('aws');
+        });
+        it('should return value if referenceIdKey is included in allReferenceTypeInfo and referenceMap is provided but value is not included in referenceMap', () => {
+            expect(getWidgetValueLabel('naver cloud', { allReferenceTypeInfo: mockAllReferenceTypeInfo, referenceIdKey: 'provider' })).toBe('naver cloud');
+        });
+        it('should return reference label if referenceIdKey is included in allReferenceTypeInfo and referenceMap is provided and value is included in referenceMap', () => {
+            expect(getWidgetValueLabel('aws', { allReferenceTypeInfo: mockAllReferenceTypeInfo, referenceIdKey: 'provider' })).toBe('AWS');
+        });
+    });
```
Comment (sulmoJ) [inline apps/web/src/services/dashboards/widgets/_helpers/__tests__/widget-value-label-helper.test.ts:55]: Thx you for the example

**Review by sulmoJ — APPROVED** [review]
LGTM!
I have a question. What are some reference stores for using `dataType`, `usageUnit` parameters?

**Review by skdud4659 — APPROVED** [review]
LGTM🤸🏼

File: apps/web/src/services/dashboards/widgets/_helpers/__tests__/widget-value-label-helper.test.ts (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/dashboards/widgets/_helpers/__tests__/widget-value-label-helper.test.ts:55]: Copilot helped me a lot :)

**Pull request merged**